### PR TITLE
Bootstrapping Prefect builds

### PR DESCRIPTION
**Pull request by schuylermartin45**
Empty commit to get the Prefect builds running. Build seems to pass on local without deltas.

### CODE REVIEW

**Comment by schuylermartin45** [comment]
....and of course I forgot to check the linter.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,7 +34,21 @@ about:
   license_family: GPL
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
  license_family: LGPL
```

Comment (cbouss) [reply recipe/meta.yaml]: And license is LGPL-2.1-or-later
https://git.gnunet.org/libmicrohttpd.git/tree/COPYING?h=v0.9.76#n7

Comment (schuylermartin45) [reply recipe/meta.yaml]: Damn, I didn't think to check that. I guess that's wrong in conda-forge as well then...
 
Is that something the linter could check for? i.e. Vet what we have against the `LICENSE` file found from the repo supplied by`dev_url`

Comment (cbouss) [reply recipe/meta.yaml]: It could, but it doesn't download upstream code at the moment.

**Review by cbouss — APPROVED** [review]

**Review by AndrewVallette — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**